source/ContentScript/util.ts

```diff
@@ -0,0 +1,126 @@
+import {ElementLocator} from '../types/zestScript/ZestStatement';
```

Comment:
All new scripts should have the standard ZAP header, as per the other scripts.

Reply:
okay